OrbitGl/SamplingReport.cpp

```diff
@@ -43,7 +43,7 @@ void SamplingReport::FillReport() {
 }
 
 void SamplingReport::UpdateReport() {
```

Comment:
The only question remains is that this method called twice on symbol update - when the request initiated and when we get data from the server. But I suppose this can be addresses in a separate PR.

Reply:
Yeah, let's keep it for another change (it doesn't seem to cause any issues at the moment).